src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -66,10 +67,12 @@ export type SingleIntegrationEvent = {
     | 'Integrations: Disabled'
     | 'Integrations: Integration Tab Clicked'
     | 'Integrations: Config Saved'
-    | 'Integrations: Plugin Add to Project Clicked';
+    | 'Integrations: Plugin Add to Project Clicked'
+    | 'Integrations: Open Upgrade Modal';
```

Comment:
@NisanthanNanthakumar The event name should closely match the event key to avoid confusion. So this should be `Integrations: Upgrade Plan Modal Opened`